TESTING.md

```diff
@@ -49,9 +49,9 @@ To generate a self signed cert and key follow these [steps](https://www.linode.c
 
 Example of creating a secret and customdomain:
 ```
-oc new-project my-project
-oc create secret tls acme-tls --cert=fullchain.pem --key=privkey.pem
-oc apply -f <(echo "
+oc create ns my-project
+oc create secret tls acme-tls --cert=fullchain.pem --key=privkey.pem -n my-project  --as backplane-cluster-admin
+oc apply --as backplane-cluster-admin -f <(echo "
```

Comment:
`--as backplane-cluster-admin` is specific to SREP and shouldn't be included in public documentation

Reply:
It is a valid point. @dustman9000, feel free to revert the changes. I'd be happy to add as an SOP or wiki doc the SRE-specific version of testing CDO.

Reply:
Valid point, but backplane specific documentation is already in place.